### PR TITLE
Changed GIF

### PR DESCRIPTION
**Pull request by ZohebMOPO**
Any more issues just let me know
img

### CODE REVIEW

File: src/components/Home/index.jsx [hunk 1]
```diff
@@ -26,7 +26,13 @@ const Home = () => {
 				<p className="home_page__sub-heading">
 					Go anonymous, Go incogly.
 				</p>
-				<img src="/images/code-image.png" alt="Code" />
+				<img
+					src="https://cdn.dribbble.com/users/37380/screenshots/10883216/zoom-party4_12fps.gif"
+					alt="landing-page"
+					height="600"
+					width="800"
+					style={{ border: "1px", borderRadius: 75 + "px" }}
```
Comment (sansyrox) [inline src/components/Home/index.jsx]: @ZohebMOPO , please don't use inline styling here. Can you shift it to the relevant scss file?

**Review by sansyrox — CHANGES_REQUESTED** [review]

File: src/components/Home/index.jsx (hunk 1 above)
Comment (sansyrox) [reply src/components/Home/index.jsx]: Also, the gif looks very nice 💯

**Comment by ZohebMOPO** [comment]
Yeah sure. I will try to take those in scss.

**Comment by ZohebMOPO** [comment]
@sansyrox I migrated the Inline CSS to SCSS. It works fine! I tested it
meet2

**Comment by sansyrox** [comment]
@ZohebMOPO , can you share the original dribbble link of the gif?

**Comment by ZohebMOPO** [comment]
https://cdn.dribbble.com/users/37380/screenshots/10883216/zoom-party4_12fps.gif

**Pull request merged**